vendor/modules.txt

```diff
@@ -2,8 +2,9 @@
 github.com/beorn7/perks/quantile
 # github.com/blang/semver v3.5.0+incompatible
 github.com/blang/semver
-# github.com/ceph/go-ceph v0.2.0
-github.com/ceph/go-ceph/errutil
+# github.com/ceph/go-ceph v0.3.0
```

Comment:
@nixpanic is this automatically added to modules.txt?

Reply:
Yes, with commands like
```
$ go mod edit -require=github.com/ceph/go-ceph@v0.3
$ go mod vendor
```